integration_tests/test_parser.py

```diff
@@ -208,6 +208,8 @@ def _setup_origin(self, snapcraft_files, repo_dir, base_dir):
     ]
 
     def test_local_wiki(self):
+        self.useFixture(fixtures.EnvironmentVariable(
+            'XDG_CACHE_HOME', os.path.join(self.path, '.cache')))
```

Comment:
I'm curious why we don't want the global cache here?

Reply:
The parser is using the cache as a temporary directory, I don't remember why, but here's a TODO related to this: https://github.com/snapcore/snapcraft/blob/5d9c6f2f7c66be4c13dc74f89c2e9fd8fd65716b/snapcraft/internal/parser.py#L59

So, when we use the real cache, we are affecting these tests in mysterious ways. To fix this, we need to finish that TODO.

Reply:
Huh, okay, fair enough.

Reply:
This is where the wiki data gets saved to, overwriting it with data from mocked/fake servers would not be great